docs/geedocs/5.2.5/answer/176738.html

```diff
@@ -257,6 +257,10 @@ <h2 id="cache_configuration">Cache configuration</h2>
 Note that the asset version cache is generally larger than the asset cache because it may contain multiple versions of the same asset.</p>
 
 Also included within <code>misc.xml</code> is an experimental option, which allows for the user to disable cache purging while projects build. This is controlled by <code>DisablePacketLevelVersionCachePurge</code> and its default value is <code>false</code>.
+
+<h2 id="overlap_calculation">Overlap Calculation</h2>
```

Comment:
Thanks for updating the docs. These changes are going to end up in 5.3.1, so that's where the documentation should go, not 5.2.5. The only way to do that right now is to have a separate PR for the docs.